src/fission.F90

```diff
@@ -102,4 +106,57 @@ function nu_delayed(nuc, E) result(nu)
 
   end function nu_delayed
 
+!===============================================================================
+! YIELD_DELAYED calculates the fractional yield of delayed neutrons emitted for
+! a given nuclide and incoming neutron energy in a given delayed group.
+!===============================================================================
+
+  function yield_delayed(nuc, E, g) result(yield)
```

Comment:
This can only be made a `pure` function if `interpolate_tab1` is also made `pure`. Then `interpolate_tab1` can only be made pure if `binary_search` and `fatal_error` are made `pure`. It seems like all the functions in `fission.F90` should be made pure, but doing so requires making several other functions in other input files to be `pure`. How about we just make an issue to go through and purify all qualifying functions?